### PR TITLE
makes stimulum not cancer

### PR DESCRIPTION
**Pull request by imsxz**
## About The Pull Request

2 line change
## Why It's Good For The Game

stimulum is the easiest thing in the game to make ever since the recent fusion change, stimulu is much too strong for how easy it is to make in a low amount of time.

## Changelog
:cl:
balance: stimulum now causes toxin damage
/:cl:

### CODE REVIEW

**Comment by Jaltsi** [comment]
hjonk

**Comment by imsxz** [comment]
i have murderboned with stimulum+carp+desword so i can legally make this PR

**Comment by imsxz** [comment]
blah blah it encourages teamwork with sci to become slimeperson so that u dont take tox damage etc etc

**Comment by TerraGS** [comment]
Slimeperson buff!

**Comment by Rowelll** [comment]
yates meador/jaltsi nerf

**Comment by Ty-the-Smonk** [comment]
the xenobio powercreep will win

**Comment by TerraGS** [comment]
Time to murderbone as a radioactive slimeperson on stimulum

**Comment by nicbn** [comment]
Now it's mostly useless, I don't see the point

**Comment by imsxz** [comment]
>mostly useless
nice bait

**Comment by dr-bee** [comment]
This is more a problem with fusion than with Stimulum isnt it?

**Comment by Ratvar** [comment]
No, it's both

**Comment by nemvar** [comment]
I member when people told me stimulum is useless due to taser removal. Never forgetti.

**Comment by necromanceranne** [comment]
Just eat some donk pockets.

**Comment by imsxz** [comment]
@dr-bee it's cancer regardless of how quickly you can do fusion, it was just less cancer when it took 20 mins to make instead of 5

**Comment by MCterra10** [comment]
Only actual atmos GODS can get enough tritium to make enough stimulum from fusion in 5 minutes, and would essentially require having a detailed mental map of exactly where you want your pipes. Fusion is gonna take at least an hour for 95% of people. I really don't see a high end gas like stim needing a nerf

**Comment by imsxz** [comment]
you dont have to be a gamer god to follow a tutorial on fusion terra dont fool yourself

**Comment by PKPenguin321** [comment]
While you're here can you put a period at the end of line 1171

**Comment by MCterra10** [comment]
I'm fully aware of that, no need to be a shit about it, @imsxz. My point is that supersaturated burns take time to create enough tritium for fusion, especially if you don't expand and optimize the incinerator chamber, and additionally it takes time to run piping, get research for stock parts, build freezers, pump co2 and plasma into a canister, get that canister up to temp and then wait for fusion to conclude. Then you need to cool your canister and filter off the probably about 30 moles of stim produced by the unoptimized fusion reaction, and pump them into an air tank. Fusion is a time intensive process, and remains so even if tutorials make the whole thing unga level easy.

**Comment by tralezab** [comment]
make sure to add the toxin damage to the description of the reagent

and i will also pester you to add a period as well

**Comment by Tlaltecuhtli** [comment]
old change was intended for when stimulum took 1h+ and there werent minecraft tutorials, both of these are false now so its fine for it to be changed to have a minor debuff to not 24/7 on stim without some extra minor effort to deal with the toxloss

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1185,7 +1185,9 @@
 	..()
 
 /datum/reagent/stimulum/on_mob_life(mob/living/carbon/M)
+	var/tox_damage = current_cycle*0.1*REM // 1 toxin damage per cycle at cycle 10
 	M.adjustStaminaLoss(-2*REM, 0)
+	M.adjustToxLoss(tox_damage, 0)
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: just do the calc in here, If you were pulling from perf the var used to save it from pulling `current_cycle` multiple times

**Comment by ExcessiveUseOfCobblestone** [comment]
This is potentially an incredibly strong buff to slimepeople who get their hands on the gas although the idea science and engineering work together is a bit silly in the current iteration.

**Comment by imsxz** [comment]
does it have more effects than just healing toxin damage on slime people? not much even causes tox to slimepeople other than stuff like charcoal, of which would purge the stimulum from their body and theyd start dying from the charcoal right after.

**Comment by ATH1909** [comment]
> does it have more effects than just healing toxin damage on slime people? not much even causes tox to slimepeople other than stuff like charcoal, of which would purge the stimulum from their body and theyd start dying from the charcoal right after.

I think slimepeople (and maybe other types of jellypeople? I don't quite recall) can gain blood (which they can then spend to split themselves) when their toxin lover trait activates.

**Comment by ExcessiveUseOfCobblestone** [comment]
We'll just call it emergent gameplay

**Pull request merged**